src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue3262.cs

```diff
@@ -36,12 +37,12 @@ public void ChangeDuringNavigating()
 		}
 
 		[Test]
-		[Category(UITestCategories.WebView)]
 		[Category(UITestCategories.Compatibility)]
-		[FailsOnAllPlatformsWhenRunningOnXamarinUITest]
+		[FailsOnMac]
```

Comment:
Needs also to update this attributes, have changed.
`[FailsOnMacWhenRunningOnXamarinUITest]`